### PR TITLE
Feat:Added Login Page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added Login Page

Screenshot from 2022-03-27 17-30-11

### CODE REVIEW

File: src/Components/Login/Login.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    height: 2rem;
+    width: 18.5rem;
+    border: 1px solid var(--gray-color);
+}
+.login-btn{
+    padding: 0.6rem 8.4rem;
+    font-weight: bold;
+    background-color: var(--blue-color);
+    border: none;
+    color: white;
```
Comment (khushijohri2001) [inline src/Components/Login/Login.css:33]: You can use var here instead of using color name directly

**Review by khushijohri2001 — APPROVED** [review]
Clean code and well-aligned with uniform indentation. Overall liked it. Good for Merge!!

File: src/Pages/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Login } from '../../Components/Login/Login'
+import { Navbar } from '../../Components/Navigation/Navbar'
+
+const LoginPage = () => {
+  return (
+    <div>
+        <Navbar/>
```
Comment (abhisheksahani-nova) [inline src/Pages/LoginPage/LoginPage.js:8]: Generally we dont add Navbar in login and signup page

**Review by abhisheksahani-nova — APPROVED** [review]
Looks subtle and good to merge

File: src/Components/Login/Login.css (hunk 1 above)
Comment (neetukumari4858) [reply src/Components/Login/Login.css:33]: okey

**Pull request merged**